integrations/gitlab/gitlab_integration/git_integration.py

```diff
@@ -121,6 +121,33 @@ class GitlabResourceConfig(ResourceConfig):
     selector: GitlabSelector
 
 
+class GitlabMemberSelector(Selector):
+
+    include_public_email: bool = Field(
+        alias="enrichWithPublicEmail",
+        default=False,
+        description="If set to true, the integration will enrich group members with public email field. Default value is false",
+    )
+    include_inherited_members: bool = Field(
+        alias="includeInheritedMembers",
+        default=False,
+        description="If set to true, the integration will include inherited members in the group members list. Default value is false",
+    )
+    # The "include bot members" flag affects both the "group" and "member" kinds.
+    # To prevent inconsistencies, the behavior or value of this parameter should be consistent for both "groups" and "members".
```

Comment:
```suggestion
    # To prevent inconsistencies, the behavior or value of this parameter should be consistent for both "groups-with-members" and "project-with-members".
```

Reply:
sorry, this is no longer the case